static/app/views/dashboards/manage/dashboardTable.tsx

```diff
@@ -128,6 +144,33 @@ function DashboardTable({
       );
     }
 
+    if (
+      column.key === ResponseKeys.ACCESS &&
+      organization.features.includes('dashboards-edit-access')
+    ) {
+      /* Handles POST request for Edit Access Selector Changes */
+      const onChangeEditAccess = (newDashboardPermissions: DashboardPermissions) => {
+        const dashboardCopy = cloneDeep(dataRow);
+        dashboardCopy.permissions = newDashboardPermissions;
+
+        updateDashboardPermissions(api, organization.slug, dashboardCopy).then(
+          (newDashboard: DashboardDetails) => {
+            onDashboardsChange();
+            addSuccessMessage(t('Dashboard Edit Access updated.'));
+            return newDashboard;
+          }
+        );
+      };
```

Comment:
Is it possible to export this function from [detail.tsx](https://github.com/getsentry/sentry/blob/4e0bd4d5943274127b0fa28bb926d0bdd3411efc/static/app/views/dashboards/detail.tsx#L632) instead?

Reply:
unfortunately not cause it's in a class component 😞

Reply:
Technically we _could_ extract this, we'd just have to pass along the dependencies whether they be from the class component or just functional component props, but I'm okay with duplicating this small function for now.

Reply:
sounds good, I can look into extracting the function in a later PR